### ISSUE TITLE
Request design review for Yellow Ribbon MVP

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#4944]
## Product Name

Yellow Ribbon MVP

## Checkpoint Stage

[ ] **Design Intent** (for early discussions about needs, priorities, concepts, research planning, and direction)

[x ] **Pre-usability testing Review** (for prototype and testing validation of best practices, alignment with va design system, and inclusive research best practices)

[ ] **Design Review** (for design work validation of best practices and alignment with the va design system)

### Where in the process is this? 

[ ] Initial concept

[ ] Initial mockup

[ x] Final mockup

## Considerations
Adding this section for additional considerations/items of note for reviewers since this is occurring async and I will not be presenting

- Content and Design (would appreciate any feedback specific to these points)
   1. Some copy edits are still pending for the intro page regarding whether or not to show eligibility info and the use of the word "Funding" and the labels indicating how many students can get YR benefit
   2.  The sticky green bar needs design refinements, and the point has been raised that this might compete w/ sticky banners sometimes on the site.  Would love feedback/input on either of these points
  3. The cards on desktop feel a little like there's too much empty space. I tried a 2-col layout with the cards, but that introduced confusion about the order they were in (especially once we add sorting or filtering), and reduced scan-ability of the information.  I have considered a list (e.g. google search results) solution, but haven't been able to resolve how to make that scannable without excessive white space.
  4.  We've learned the GIDS API doesn't currently have sorting available. We are leaving out sorting and filtering for now, and will ask participants questions to learn more about what will be useful to them

- The prototypes:
 1. Key differences to review on mobile  are the breadcrumbs, the way users access search criteria on the results page, and how to view different data points on the comparison screen
 2. The prototypes have hotspots that will be used with participants. Clicking anywhere on the InVision screen will trigger a blue flash on the hotspots so you know where to click.
3. A clue on the clickable hot spots: on the results screen, the participants will be asked to imagine they're interested in studying art and selecting schools where they could do that with YR benefits.

- Finally, the conversation guide and research plan have not been updated to reflect recent changes. We will have a segment of users go through tasks on current YR content, and another go through this prototype.  Working on that next.

## Design links or mockups
[Mobile](https://vsateams.invisionapp.com/share/XBVON0YMJ4Y)
[Desktop](https://vsateams.invisionapp.com/share/X5VOMBG2YP4)

## Point of contact

Liz Lantz

**Comment by andaleliz** [comment]
I'm creating this ticket now in hopes of getting time reserved on everyone's calendar - the prototype isn't done now, but I plan to have it completed by Wednesday.  My pilot test is scheduled for 1/23 and we are hoping to begin sessions with participants on 1/27.

This will be a hi-fi version of what you've already seen, using the design system wherever possible (there may be 2 non-ds components) so I don't anticipate a big departure from what you've seen.

**Comment by meganhkelley** [comment]
Thanks, Liz! I'm coordinating with the practice area reviewers on this and will work to get something scheduled. 

Would it be possible to have the prototype done 2 days before the scheduled meeting, to give reviewers time to prepare feedback before we meet? Would that change the preferred dates/times you noted in your ticket?

**Comment by sshein** [comment]
I did tell Liz that the "pre-usability test" review isn't required by VSA at the moment, so if this doesn't work out it's ok. You all already reviewed her wireframes not very long ago, in the "design intent checkin."  In general would prefer not to change or hold up her timeline since she already reached out to Lauren and is getting participants scheduled.

**Comment by sshein** [comment]
obviously after the usability test there will be a chance to make edits and get more feedback, which can be incorporated with the users' feedback

**Comment by andaleliz** [comment]
Hi reviewers!  I've become aware that there is a opportunity for greater alignment on my team in terms of what we're building, and am postponing the usability testing until we can get on the same page.  I'm meeting with them tomorrow, but some changes to our MVP may come as a result of the conversation.

To make sure you see the latest plan from our team, I'd like to postpone this review, but leave the issue open. 

I'll keep you posted.

**Comment by AnneHurley** [comment]
@peggygannon @meganhkelley  @andaleliz  This is the ticket we created specifically for content related to the YR tool: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/4881
The latest doc, including the one with Randi's copy edits, is on there.

**Comment by andaleliz** [comment]
Hey all, this is ready for your review.  I'm planning on telling Perigean to resume recruiting and hope that we can schedule usability tests starting Wed next week.

How will I know once the review is complete?

**Comment by mnorthuis** [comment]
@andaleliz 
I see that you are segmenting users to go through either the the old or this new experience, does  your conversation guide have all users showing where/how they would access this information?  That task can give insight into where this tool should be located or linked to. 

FYI, in your prototype header, the down carat that should be next to the "VA Benefits and Health Care" top nav option is overlapping the label in the middle of the word "Health". 

No other IA feedback at this time, will complete final IA after usability is complete.  I created a placeholder issue for that step here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/5192

**Comment by andaleliz** [comment]
Thank you Mikki! We will have users show where/how they'd access the information.  I had started working on adding that in yesterday and your comment makes me feel extra confident about this change!

We have an issue with our pattern library versioning causing that issue with the down carat; I went ahead and manually fixed it my files. Thanks for spotting that.

**Comment by 1Copenut** [comment]
@andaleliz Are there any accessibility changes or updates @jenstrickland and I should review?

**Comment by andaleliz** [comment]
@1Copenut this is a hi-fi version of the other prototype you reviewed, so it has changed quite a bit since the last time you saw it.  Jen opened this issue, feel free to leave any additional comments or feedback you have: https://github.com/department-of-veterans-affairs/va.gov-team/issues/5324

**Comment by 1Copenut** [comment]
@andaleliz Thank you for approaching with mobile and desktop views. LMK if any of my comments don't make sense or need clarification.

**Comment by andaleliz** [comment]
@1Copenut Thanks for the feedback! I'm pretty clear on all of it.  I love using InVision for comments, but the developers don't have access to it and I don't want anything to get lost in translation.  Your comments about the sticky footer + focus areas are particularly relevant to their work.

My understanding of this process was that each reviewer would create an issue under this epic and assign it to me with their comments and feedback. If that's the case, would you mind putting yours in there and adding Nick + Kelson as assignees? That way they can ask follow up questions directly and have a record for their work.

cc @meganhkelley for awareness as you refine this process and ensure all the right feedback goes to the right people.

If that's not the case, no prob - I'll figure out some other way to capture this for them.

**Comment by 1Copenut** [comment]
@andaleliz sure thing, I"ll get an issue added to this epic after a few items this afternoon

**Comment by andaleliz** [comment]
@jenstrickland, I saw your post in our slack channel about this.  I'm not sure what, if anything needs to happen before we can close this out. CC @meganhkelley  in case she can help.  Not clear on this process.

**Comment by jenstrickland** [comment]
@andaleliz I'm nor sure which post this refers to. Is it this one? https://dsva.slack.com/archives/C52CL1PKQ/p1581372141009600

@jenniferlee-dsva @ncksllvn @kelsonic @1Copenut and I had a meeting to discuss the smooth scroll yesterday, and why we should not implement smooth scroll based on guidance from Deque and Level Access consultants. Jen asked that I review the accessibility tickets assigned to your team, especially those in the icebox, to ensure they were all still viable. That's what I reviewed and what the post was about. I'm not sure where the team is with the feedback from the YR design review and sidenav work, and consider those documentation for the team to use in their work. Some teams comb the review content and document decisions made on the review ticket in a wrap-up comment before they close the ticket.
